packages/table/src/Row/Row.styles.ts

```diff
@@ -0,0 +1,123 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import {
+  focusRing,
+  hoverRing,
+  transitionDuration,
+} from '@leafygreen-ui/tokens';
+
+export const rowBaseStyles = css`
+  transition: ${transitionDuration.default}ms ease-in-out;
+  transition-property: all;
```

Comment:
Maybe set this to only the properties that you need. When you toggle between light and dark mode the background colors linger for a while.

Reply:
Turns out this isn't even needed since the Cell is what ended up handling the transitions!